paket.dependencies

```diff
@@ -5,3 +5,5 @@ nuget Funcky ~> 1.2.0
 nuget Microsoft.NET.Test.Sdk ~> 16.2.0
 nuget xunit ~> 2.4.0
 nuget xunit.runner.visualstudio ~> 2.4.0
+nuget Equals.Fody ~> 1.9.5
+nuget Fody ~> 4.0
```

Comment:
Use the latest version and make the dependencies private.
See:
https://github.com/messerli-informatik-ag/file-opening-builder/blob/c3abf3f156006e9737430192c66e5d6f4e346134/paket.dependencies#L5-L6

Reply:
https://github.com/messerli-informatik-ag/routing/pull/6/commits/fed414c46866844f7309b2214beb1e5a38d461ed